### ISSUE TITLE
Tensorflow Compatibility Issue 

### ISSUE THREAD

**Issue by MichealSBingham** [xIris-Corp/RoadAPI#1]
One of the issues you're going to encounter here is that this does not work with tensorflow 2.0+ (because of a certain deprecated function that's used) 

This version is meant to be used with I think Tensorflow 1.6 but If I remember correctly any version less than 2.0 will do so your options are either: 

1. Find a way to install an older tensorflow version (can compile a binary file, or use anaconda to install an older version) 
2. Find the errors and convert it to be used with tensorflow 2.0  (this we will eventually _have_ to do but not right now. Not sure how long this solution may take to implement but I don't think it's too much trouble actually) 

Be sure to document which solution you used here and then commit your changes as we will need to perform the same tensorflow installation steps when we upload it to our virtual machines.

**Comment by ericsong01** [comment]
What's worked for me so far is using conda to install v1.13 of tensorflow (since v1.13 supports Python 3.7, past versions did not). 
I ran the following inside my RoadAPI virtualenv to install it `conda install -c conda-forge tensorflow=1.13`. 
For now I'll move on to the matplotlib issue, but if I find a good fix for that I'll come back and see if there's a fix for tensorflow v2.0+

**Comment by ericsong01** [comment]
I updated to tensorflow v2.4.0 and was able to get the Detection code running by changing some of the outdated tf1.0+ code. 
I followed the below steps to upgrade to tf v.2.4:
(I initially made a conda virtualenv that used Python 3.7 but I found that v2.4 works with Python 3 and 3.6+
Also, using conda to install/update wasn't updating tf to v2.0+ so I installed it through pip in the conda virtualenv through the steps below)

1. `conda install pip`
2. `pip install tensorflow`
3. `pip install --upgrade tensorflow`
